src/web/View.tsx

```diff
@@ -359,7 +359,9 @@ export class View extends ViewBase<RX.Types.ViewProps, RX.Types.Stateless, RX.Vi
             onMouseMove: this.props.onMouseMove,
             // Weird things happen: ReactXP.Types.Touch is not assignable to React.Touch
             onTouchStart: this.props.onResponderStart as React.HTMLAttributes<any>['onTouchStart'],
+            onTouchStartCapture: this.props.onTouchStartCapture as React.HTMLAttributes<any>['onTouchStartCapture'],
```

Comment:
What happens on native platforms?

Reply:
This code acts only for web. I am not sure to understand.

Reply:
If you're exposing a common prop, a RN implementation should also be provided. If not possible, it should be documented to noop and have good justification why it's needed

Reply:
Ok, I will add some context and test first.